data/coalition/coalition missions.txt

```diff
@@ -3211,3 +3211,585 @@ mission "Heliarchs Buy Jump Drive Mission"
+		payment 750000
+		conversation "anna invites you to book signing"
+
+
+conversation "anna invites you to book signing"
+	action
+		log "People" "Anna Fangvin" `Anna is an ambidextrous science fiction writer. A struggling author in human space, she found success upon publishing her novel 'Beyond the Hyperdrive' in the Coalition. Her book is a bestseller in the Kimek region.`
+		log "Helped somebody named Anna Fangvin get her book published in Coalition space. Apparently, it's a bestseller now. The book was 'Beyond the Hyperdrive', something about humans stealing jump drives from the Quarg."
+		event "beyond the hyperdrive payment" 40
+		set "beyond the hyperdrive news can show now"
+		outfit "Beyond the Hyperdrive" 1
+	`The moment you land, Anna arrives, very happy to see you. She grabs your shoulders with both arms and in an excited tone explains that her book has been approved for publication.`
+	`	"They're publishing 'Beyond the Hyperdrive' today, and there's a full release event where I'll be signing copies! Thanks for everything you've done, <first>! Do you want to join me during the book signing?"`
+	choice
+		`	"I've been with you through the whole process. Count me in!"`
+		`	"I'd be happy to!"`
+	`	"Great! Let's walk to the bookstore and get ready! The staff have already prepared seats for both of us."`
+	`	You and Anna walk into the massive bookstore at the spaceport. There are already several Heliarchs present, their Enforcer Riot Staffs in place for crowd control.`
+	`	A Kimek, dressed in what looks like the bookstore's uniform, takes out a loudspeaker with multiple handholds and begins announcing the release of 'Beyond the Hyperdrive'. Within minutes, the bookstore has become crowded, with a long queue forming between the cashiers and your table, as Anna signs every copy brought to her.`
+	`	Occasionally, Anna alternates between her left and right hand to sign books whenever the other is tired. For hours, this continues. You tried to keep count of how many copies she signed, but gave up when it reached six hundred.`
+	choice
+		`	(Say nothing.)`
+			goto continue
+		`	"Are you left- or right-handed?"`
+	`	"I was originally born left-handed, but as a child I chose to learn how to use my right hand, because I thought being able to use both hands would be cool."`
+	label continue
+	`	The signing continues for several more hours. Eventually, the same Kimek in the bookstore uniform announces that the store is closing soon.`
+	`	As the store closes and the queue shortens, Anna signs the last Kimek's copy. With that, the event is over. Anna walks with you back to your ship to say goodbye.`
+	`	"My book succeeded beyond my wildest dreams, and it was all because of you, <first>!" she says, almost in a teary voice. She pulls out a credit chip. "Here, <payment>. You've done so much for me and my career, and you deserve it. Additionally, you will be receiving a salary of 100 credits daily, a small but significant portion of my royalties."`
```

Comment:
Took out the salary line.
```suggestion
	`	"My book succeeded beyond my wildest dreams, and it was all because of you, <first>!" she says, almost in a teary voice. She pulls out a credit chip. "Here, <payment>. You've done so much for me and my career, and you deserve it."`
```